### PR TITLE
refs #21350 - allow restricting ciphers and protocols

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
@theforeman-bot is unhappy, but this issue spans many different repos.

File: templates/router/ssl_profile.conf.erb [hunk 1]
```diff
@@ -6,5 +6,11 @@ ssl-profile {
 <% unless [nil, :undefined, :undef, ''].include?(@password) -%>
     password: <%= @password %>
 <% end -%>
+<% unless [nil, :undefined, :undef, ''].include?(@ciphers) -%>
+    ciphers: <%= @ciphers %>
+<% end -%>
+<% unless [nil, :undefined, :undef, ''].include?(@protocols) -%>
+    protocols: <%= @protocols.join(" ") %>
```
Comment (sean797) [inline templates/router/ssl_profile.conf.erb]: The current version of `qpid-dispatch-router` in epel doesn't support this parameter. I assume it is coming in a future release?

Comment (stbenjam) [reply templates/router/ssl_profile.conf.erb]: Yes, sorry.    It's already been merged, but waiting for an answer from the qpid devs about when we'll see it in EPEL.

Comment (ekohl) [reply templates/router/ssl_profile.conf.erb]: Since the default is undef, I'd be OK with including this patch.

**Comment by stbenjam** [comment]
Ok thanks!

**Pull request merged**